lib/Backends/OpenCL/OpenCLDeviceManager.cpp

```diff
@@ -114,6 +115,12 @@ void OpenCLDeviceManager::addNetworkImpl(const Module *module,
       readyCB(module, ResultCode::Failed);
       return;
     }
+
+    if (func.second->getCompileBackendKind() != BackendKind::OpenCL) {
+      llvm::errs() << "Failed to add network: function " << func.first
+                   << " is not an Function.\n";
```

Comment:
Just want to double check: the error message means "not an function" or "not a correct backend function" ?

Reply:
This should say "not an OpenCL function" but looks like i deleted the important word, thanks!